src/test/java/org/opensearch/geospatial/GeospatialRestTestCase.java

```diff
@@ -61,9 +58,9 @@ protected static void deletePipeline(String name) throws IOException {
     }
 
     protected static void createIndex(String name, Settings settings, Map<String, String> fieldMap) throws IOException {
-        XContentBuilder xContentBuilder = XContentFactory.jsonBuilder().startObject().startObject(Feature.PROPERTIES_KEY);
```

Comment:
Why hard code these?

Reply:
Though the value of constant is what i wanted, constant name is not appropriate for the value. I created new constant (duplicate value) with valid name. Do you have any suggestion for how to handle this case? i am not convinced of this method but don't know what is the alternative.

Reply:
Ah oh I mixed this up. I thought this was a source file. Im okay with either approach.